t/05-fullstack-scalability.t

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env perl -w
```

Comment:
I guess the files are not even marked as executable anymore so probably we should delete it from all test files